lib/v2/sehuatang/user.js

```diff
@@ -0,0 +1,104 @@
+// 导入必要的模组
+const got = require('@/utils/got'); // 自订的 got
+const cheerio = require('cheerio'); // 可以使用类似 jQuery 的 API HTML 解析器
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const config = require('@/config').value;
+
+const baseUrl = 'https://sehuatang.org/';
+
+module.exports = async (ctx) => {
+    // 从Url参数中获取uid
```

Comment:
https://github.com/DIYgod/RSSHub/blob/9bdba060c7962d734745c8d6b9264da9a532b51d/lib/routes/disqus/posts.js#L5-L7
Similarly, you can throw an error if the cookie is not set.